module/bdev/lvol/vbdev_lvol.c

```diff
@@ -1357,6 +1358,43 @@ vbdev_lvol_rename(struct spdk_lvol *lvol, const char *new_lvol_name,
 	spdk_lvol_rename(lvol, new_lvol_name, _vbdev_lvol_rename_cb, req);
 }
 
+static void
+_vbdev_lvol_set_xattr_cb(void *cb_arg, int lvolerrno)
+{
+	struct spdk_lvol_req *req = cb_arg;
+
+	if (lvolerrno != 0) {
+		SPDK_ERRLOG("Setting xattr failed\n");
+	}
+
+	req->cb_fn(req->cb_arg, lvolerrno);
+	free(req);
+}
+
+void
+vbdev_lvol_set_xattr(struct spdk_lvol *lvol, const char *name,
```

Comment:
It seems that this function acts only as a wrapper of `spdk_lvol_set_xattr` without adding nothing, maybe it is better to call directly the lvol function in the RPC. The same for the get function below

Reply:
Yeah, but following the convention might be better?

Reply:
For example:

- `rpc_bdev_lvol_inflate` calls directly `spdk_lvol_inflate`
- `rpc_bdev_lvol_decouple_parent` calls directly `spdk_lvol_decouple_parent`

But `rpc_bdev_lvol_set_read_only` calls `vbdev_lvol_set_read_only` that is only a wrapper without adding nothing, like your function. 
I prefer the first approach, but feel free to follow the way you like more.